cluster/ringmaster/lz-Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM nvidia/opencl:runtime-ubuntu16.04
+RUN apt-get update --no-upgrade -yq && \
+    apt-get install --no-upgrade -yq curl git clinfo cmake g++ libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev qtbase5-dev python-virtualenv lsb-release
+
+RUN mkdir /app && cd /app && \
+    git clone https://github.com/gcp/leela-zero.git --branch next . && \
+    git submodule update --init --recursive && \
+    sed -i 's!//#define USE_TUNER!#define USE_TUNER!' src/config.h && \
+    mkdir build && cd build && \
+    cmake .. && cmake --build . -- -j 8
+    # ./tests
+
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+    apt-get update -y && apt-get install google-cloud-sdk -y
+
+
+WORKDIR /
+RUN virtualenv -p /usr/bin/python2 mg_venv
+RUN . mg_venv/bin/activate \
+  && pip install gomill
+# ringmaster now available at /mg_venv/bin/ringmaster
+
+RUN cp /app/build/leelaz /leelaz
+
+#gcp plz! this should be a flagfile.
+ADD staging/p100-lz-tuning /root/.local/share/leela-zero/leelaz_opencl_tuning
```

Comment:
how are we going to get visibility if this breaks? just assume that you and I are looking at some of the log files?